apps/condo/next.config.js

```diff
@@ -23,6 +23,8 @@ const defaultLocale = conf.DEFAULT_LOCALE
 const insuranceAppUrl = conf['INSURANCE_APP_URL']
 const JivoSiteWidgetId = conf['JIVO_SITE_WIDGET_ID']
 const TinyMceApiKey = conf['TINY_MCE_API_KEY']
+const featureToggleApiUrl = conf['FEATURE_TOGGLE_API_URL']
```

Comment:
May be move to a single key with json content? Like `FEATURE_TOGGLE_CONFIG='{"url": ..., "apiKey":...}'`